src/partials/_footer.hbs

```diff
@@ -0,0 +1,19 @@
+<footer class="footer">
+	<div class="footer__container">
+		<div class="footer__links">
+			<a href="#" class="footer__link">Privacy Policy</a>
+			<a href="#" class="footer__link">Terms &amp; Conditions</a>
+			<p class="footer__text">A Modern Eden &copy; 2016</p>
+		</div>
+
+		<div class="footer__site__logo">
+			<a href="#" class="footer__logo icon-logo-white">A Modern Eden</a>
+		</div>
+
+		<div class="footer__icons">
+			<a href="tel:1-234-567-8900" class="footer__icon icon-phone"></a>
+			<a href="#" class="footer__icon icon-facebook"></a>
+			<a href="https://twitter.com/amoderneden?lang=en" class="footer__icon icon-twitter"></a>
+		</div>
+	</div>
+</footer>
```

Comment:
Great job on naming your classes here using the BEM convention. Makes this HTML component very readable.